src/Common/tests/System/Collections/IEnumerable.Generic.Serialization.Tests.cs

```diff
@@ -15,6 +15,13 @@ public abstract partial class IEnumerable_Generic_Tests<T> : TestBase<T>
         public void IGenericSharedAPI_SerializeDeserialize(int count)
         {
             IEnumerable<T> expected = GenericIEnumerableFactory(count);
+
+            // Not all IEnumerables are intended to be Serializable
+            if(!expected.GetType().IsSerializable)
+            {
+                return;
+            }
```

Comment:
We already have:
https://github.com/morganbr/corefx/blob/eec6217de02d28f0060c16a0b0e34a530269eb6b/src/Common/tests/System/Collections/IEnumerable.NonGeneric.Tests.cs#L65
for the non-generic counterpart.  We should ideally keep them similar.  But this is ok for now.